task4/style.css

```diff
@@ -0,0 +1,510 @@
+    width: 100%;
+}
+
+.left-column {
+    display: flex;
+    justify-content: flex-end;
+    align-items: flex-end;
+    flex-direction: column;
+    width: 50%;
+}
+
+.right-column {
+    display: flex;
+    justify-content: flex-start;
+    align-items: flex-start;
+    flex-direction: column;
+    width: 50%;
+}
+
+.box-main-1 {
+    width: 344px;
+    height: 303px;
+    background: #FAFAFA;
+    box-shadow: 0px 0px 2px rgba(0, 0, 0, 0.12), 0px 2px 2px rgba(0, 0, 0, 0.24);
+    border-radius: 2px;
+    margin-right: 29px;
+    margin-top: 82px;
+}
+
+.box-1 {
+    width: 344px;
+    height: 175px;
+    background: #009688;
+    padding-top: 1px;
+}
+
+.box-1-text {
+    font-family: "Roboto", sans-serif;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 14px;
+    line-height: 24px;
+    margin-left: 18px;
+    margin-top: -3px;
+}
+
+
+.box-main-2 {
+    width: 344px;
+    height: 224px;
+    margin-top: 82px;
+    margin-left: 27px;
+    background: #FAFAFA;
+    box-shadow: 0px 0px 2px rgba(0, 0, 0, 0.12), 0px 2px 2px rgba(0, 0, 0, 0.24);
+    border-radius: 2px;
+}
+
+.box-2 {
+    width: 344px;
+    height: 176px;
+    background: #00BBD3;
+    padding-top: 1px;
+}
+
+.box-2-text {
+    width: 161px;
+    height: 24px;
+    font-family: "Roboto";
+    font-style: normal;
+    font-weight: normal;
+    font-size: 24px;
+    line-height: 24px;
+    color: #FFFFFF;
+}
+
+.btn-container-1 {
+    display: flex;
+    margin-top: 8px;
+    margin-left: 1px;
+    padding-left: 11px;
+}
+
+.btn-1 {
+    text-transform:uppercase;
+    margin-top: 9.5px;
+    margin-left: 8px;
+    margin-bottom: 8px;
+    width: 45px;
+    height: 17px;
+    font-family: "Roboto";
+    font-style: normal;
+    font-weight: 500;
+    font-size: 14px;
+    line-height: 16px;
+    color: #009688;
+}
+
+.btn-2 {
+    text-transform:uppercase;
+    margin-top: 9.5px;
+    margin-bottom: 8px;
+    margin-left: 25px;
+    width: 86px;
+    height: 17px;
+    font-family: "Roboto";
+    font-style: normal;
+    font-weight: 500;
+    font-size: 14px;
+    line-height: 16px;
+    color: #009688;
+}
+
+.divider {
+    margin-top: 22px;
+    width: 344px;
+    height: 0.1px;
+    border: 0.1px solid rgba(0, 0, 0, 0.12);
+}
+
+.h2-box1 {
+    margin-top: 138px;
+    margin-left: 18px;
+}
+
+.h2-box2 {
+    margin-top: 138px;
+    margin-left: 18px;
+}
+
+.buttons-box2-img {
+    width: 119px;
+    height: 24px;
+    margin-top: 5px;
+    margin-right: 16px;
+    margin-left: 208px;
+}
+
+.buttons-box3 {
+    width: 119px;
+    height: 24px;
+    margin-top: 5px;
+    margin-left: 24px;
+}
+
+.box-main-3 {
+    width: 168px;
+    height: 216px;
+    background: #FAFAFA;
+    box-shadow: 0px 0px 2px rgba(0, 0, 0, 0.12), 0px 2px 2px rgba(0, 0, 0, 0.24);
+    border-radius: 3px;
+    margin-top: 25px;
+    margin-left: 28px;
+}
+
+.box-3 {
+    width: 168px;
+    height: 168px;
+    background: #3E50B4;
+}
+
+.h2-box3 {
+    margin-top: -61px;
```

Comment:
if we need to use negative margins in such simple layout - we probably do something wrong